### DISCUSSION TITLE
Add fedora package for wagtail

### DISCUSSION THREAD

**Discussion by Itotutona** [wagtail/wagtail#7582] [Ideas]
I want to create a package for wagtail in fedora.

When this package will be create, we can use:
`dnf install python-wagtail`

**Comment by thibaudcolas** [comment]
I’d be interested to hear more about the value of this! Would that be only for the `wagtail start <my_site>` command? Or would you also use it to get the Wagtail version a site runs with?

It’s pretty common for Wagtail sites to also use other packages from PyPI rather than just Wagtail and nothing else – wouldn’t installing from Fedora’s package manager make that harder?

**Comment by RealOrangeOne** [comment]
The main reason to ship python packages with distros is so they can be consumed by other applications. Are you imagining that a wagtail site also ship in these package repositories? Whilst it may be useful for `wagtail start`, you'll be installing wagtail itself in a virtualenv almost immediately after, so following the regular installation instructions of creating the virtualenv first and running from there is likely much easier.

Additionally, as far as i'm aware it's more something the repository maintainers deal with when there's a need for packages to be included in the repositories, rather than something a project submits to necessarily. Given wagtails usage, I'd be surprised if they want the maintenance burden of adding it.

Interestingly, it looks like FreeBSD does ship wagtail in its package repos (https://pkgs.org/search/?q=wagtail) but not much else https://pkgs.org/search/?q=python3-wagtail / https://pkgs.org/search/?q=python-wagtail.
